locationsyncservice/build.gradle

```diff
@@ -17,6 +17,25 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    buildTypes {
```

Comment:
do we need this in the module?

Reply:
When I am building TestApplication module with QA BuildType , I am getting this error

![image](https://user-images.githubusercontent.com/38068550/77148895-76538a80-6ab6-11ea-8018-b5b53fa5f92c.png)

Therefore, Have to add similar build Type in Other module as well .